lang/en/typeshed/stdlib/microbit/audio.pyi

```diff
@@ -28,6 +28,14 @@ def is_playing() -> bool:
     :return: ``True`` if audio is playing, otherwise ``False``."""
     ...
 
+def sound_level() -> int:
+    """Returns the average intensity of the sound played.
+
+    Example: ``audio.sound_level()``
+
+    :return: A number between 0 and 254, being the average intensity of the sound played from the most recent chunk of data."""
+    ...
+
```

Comment:
WIP - To be updated once there are micropython docs

Reply:
Is this blocked on something needed to be updated in https://github.com/bbcmicrobit/micropython/ ?
Also, should `254` be `255`?

Reply:
Ah, just saw the comment on the other PR, my answer in:
- https://github.com/bbcmicrobit/micropython/pull/791#discussion_r1549521004

Reply:
We'll replace these docs with the ones you write on the docs PR (or remove them if we're not shipping **audio**.sound_level). The range came from the comment in the source code but does seem a little odd.

Reply:
Doc has been updated here: https://github.com/bbcmicrobit/micropython/pull/791/commits/87d7f75327d84afe790fdea8a88f349fdd373fe7

Reply:
Updated here: https://github.com/microbit-foundation/micropython-microbit-stubs/pull/97/commits/70578370697bcf25df0aa857184efcfe2f8b2c52